adapters/liftoff/liftoff.go

```diff
@@ -0,0 +1,180 @@
+package liftoff
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const SupportedCurrency = "USD"
+
+type adapter struct {
+	Endpoint string
+}
+
+type prebid struct {
+	Debug bool `json:"debug"`
+}
+
+type prebidExt struct {
+	Prebid prebid `json:"prebid"`
+}
+
+type liftoffImpressionExt struct {
+	*adapters.ExtImpBidder
+	// Ext represents the vungle extension.
+	Ext openrtb_ext.ImpExtLiftoff `json:"vungle"`
+}
+
+/* Builder */
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	return &adapter{Endpoint: config.Endpoint}, nil
+}
+
+/* MakeRequests */
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var requests []*adapters.RequestData
+	var errs []error
+	requestCopy := *request
+	for _, imp := range request.Imp {
+		// check if video ad
+		if imp.Video == nil {
+			errs = append(errs, fmt.Errorf("liftoff adapter handles video ads only"))
+			continue
+		}
```

Comment:
This check is not necessary. Prebid Server will not send your adapter requests for media types not defined in the yaml file. Please remove this check.

Reply:
Got it.